clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -162,8 +167,11 @@ public void start(final ServiceProvider<Service> serviceProvider) {
     } catch (RuntimeException e) {
       entityFactory = null;
       try {
-        clusterConnection.close();
+        Connection conn = clusterConnection;
```

Comment:
What triggered these connection closing changes?

And by looking at them ... and their duplication ... I can tell you a method wants to be extracted 😉